src/tests/panel.test.ts

```diff
@@ -0,0 +1,208 @@
+
+        expect(response).toBeInstanceOf(Error);
+    });
+});
+describe('_getPanelController', () => {
+    beforeEach(() => {
+        sequelizeMock = {} as jest.Mocked<Sequelize>;
+    });
+
+    test('If the panel exists, it should be returned', async () => {
+        const panelData = {
+            image:        '../../superCoolImage.png!',
+            index:        0,
+            panel_set_id: 1
+        };
+
+        (panelService.getPanel as jest.Mock).mockReturnValue(() => Promise.resolve(panelData));
+
+        const response = await _getPanelController(sequelizeMock)(1);
+
+        expect(response).toEqual(panelData);
+    });
+
+    test('If the panel does not exist, it should return undefined', async () => {
+        (panelService.getPanel as jest.Mock).mockReturnValue(() => Promise.resolve(undefined));
+
+        const response = await _getPanelController(sequelizeMock)(10);
+
+        expect(response).toBeUndefined();
+    });
+
+    test('If an error occurs, it should return the error', async () => {
+        (panelService.getPanel as jest.Mock).mockReturnValue(() => Promise.reject(new Error('Some error getting panel')));
+
+        const response = await _getPanelController(sequelizeMock)(1000);
+
+        expect(response).toBeInstanceOf(Error);
+    });
+});
+describe('_getPanelsFromPanelSetIDController', () => {
+    beforeEach(() => {
+        sequelizeMock = {} as jest.Mocked<Sequelize>;
+    });
+
+    test('If the panels exists, they should be returned', async () => {
+        const panelData = [
+            {
+                id:    1,
+                image: '../../superCoolImage.png',
+                index: 0,
+            },
+            {
+                id:    2,
+                image: '../../superCoolImage.png',
+                index: 1,
+            },
+            {
+                id:    3,
+                image: '../../superCoolImage.png',
+                index: 2,
+            }
+        ];
+
+        (panelService.getPanelsFromPanelSetID as jest.Mock).mockReturnValue(() => Promise.resolve(panelData));
+
+        const response = await _getPanelsFromPanelSetIDController(sequelizeMock)(1);
+
+        expect(response).toEqual(panelData);
+    });
+
+    test('If panel set has no panels, it shoudl return []', async () => {
+        (panelService.getPanelsFromPanelSetID as jest.Mock).mockReturnValue(() => Promise.resolve([]));
+
+        const response = await _getPanelsFromPanelSetIDController(sequelizeMock)(10);
+
+        expect(response).toEqual([]);
+    });
+
+    test('If an error occurs, it should return the error', async () => {
+        (panelService.getPanelsFromPanelSetID as jest.Mock).mockReturnValue(() => Promise.resolve(new Error('Some error getting panels')));
+
+        const response = await _getPanelsFromPanelSetIDController(sequelizeMock)(1000);
+
+        expect(response).toBeInstanceOf(Error);
+    });
+});
+describe('_createPanelController', () => {
```

Comment:
The descriptions of each test are a bit confusing, but you should only have 4 tests.
1. if panel set it null, throw an error
2. if panel set exists, return the results from `updatePanel`
3. If pansel set doesn't exist, return the results from `createPanel`
4. If an error is caught, return it

I recommend making the mocking results for `createPanel` and `updatePanel` to be different so you can verify that the controller is returning the right value